src/loaders/database.js

```diff
@@ -2,8 +2,18 @@ import { Sequelize } from 'sequelize';
 
 const sequelize = new Sequelize('postgres://postgres:password@localhost:5432/postgres');
 
+const db = {};
+
+db.Sequelize = Sequelize;
+db.sequelize = sequelize;
+
+db.user = require('../models/user.model')(sequelize, Sequelize);
```

Comment:
Requires inside code not a good style. Also you are using imports so, it's better to import first